### ISSUE TITLE
Accessibility: PropertyGrid sub-level item expanded/collapsed state is not correct

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#3099]
* .NET Core Version: 

.NET Core 5.0

* Have you experienced this same bug with .NET Framework?: 

Yes

**Problem description:**

Property Grid sub-level item accessibility: Narrator announces collapsed when Font Property node is expanded, `ExpandCollapse.ExpandCollapseState` value is `Collapsed(0)`.

**Expected behavior:**
Collapsed Property Grid sub-level items should be announced as collapsed and Inspect should show collapsed state for these items.
Expanded Property Grid sub-level items should be announced as expanded and Inspect should show expanded state for these items.

**Minimal repro:**
1. Create WinForms app and add PropertyGrid control and Button to the form.
2. Set PropertyGrid's SelectedObject to Button.
3. Run the app and review the Font property of the button in PropertyGrid using Inspect: expand and collapse the Font property and review its collapsed/expanded state.

![Inspect](https://user-images.githubusercontent.com/23213980/79792675-fab85800-8357-11ea-8be6-5626ef339929.png)
![Announcement](https://user-images.githubusercontent.com/23213980/79792699-01df6600-8358-11ea-8fe4-a851da9d790e.png)

**Comment by Olina-Zhang** [comment]
Verified this issue in .NET Core 5.0.100-preview.5.20269.19, it has been fixed. PropertyGrid sub-level item expanded/collapsed state is correct now. Please see following screenshot for Inspect and Narrator behavior:
![image](https://user-images.githubusercontent.com/26474449/82409029-6ef13300-9a9f-11ea-9c1a-9e7abe305bcb.png)
